src/test/java/com/training/services/tests/ProductsTest.java

```diff
@@ -151,4 +152,26 @@ public void testShorexOfferingDateValidate() throws ParseException {
 
         return dateMap;
     }
+
+    /**
+     * Checking if inside product type, product type name is shore excursion or aquatics.
+     */
+    @Test
+    public void testShorexProductTypeNameValidate() {
+        response = new RestEngine().getResponseGet(baseURL, headerMap, queryParam)
+                .as(Response.class);
+
+        Assertions.assertThat(response.getStatus()).isEqualTo(200).as("Json response status is not 200");
+
+        List<Products> products = response.getPayload().getProducts();
+
+        for (int index = 0; index < products.size(); index++) {
+            softAssert.assertThat((products.get(index).getProductType().getProductTypeName().equalsIgnoreCase("Shore Excursion")
+                    ||
+                    products.get(index).getProductType().getProductTypeName().equalsIgnoreCase("Aquatics")))
+                    .as("Product Id = " +
+                            products.get(index).getProductID() + " product type name is not shore excursion or aquatics")
```

Comment:
lets enhance readability : 
create a var : ` ProductTypeName productTypeName = products.get(index).getProductType().getProductTypeName()`
and then use it in assert